test/SoulLinker.test.ts

```diff
@@ -111,7 +111,7 @@ const signMintCreditScore = async (
     },
     // Types
     {
-      MintCreditScore: [
+      Mint: [
```

Comment:
this will create a breaking change, the SDK uses MintCreditScore for all the credit scores